domains/measurements/Measurements.API/API/Services/IDataSyncService.cs

```diff
@@ -0,0 +1,11 @@
+using API.Models;
+using EnergyOriginAuthorization;
+
+namespace API.Services;
+
+public interface IDataSyncService
+{
+    Task<IEnumerable<Measurement>> GetMeasurements(AuthorizationContext context, string gsrn, DateTime utcDateTime, DateTime dateTime, Aggregation aggregation);
```

Comment:
I am guessing which datetime is which, but?:

```suggestion
    Task<IEnumerable<Measurement>> GetMeasurements(AuthorizationContext context, string gsrn, DateTime dateFrom, DateTime dateTo, Aggregation aggregation);
```